.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["es2015"],
```

Comment:
I'd advise defining the babel config in the `package.json` to avoid clutter, and to define only the needed transforms rather than the whole `es2015` preset, to ease depreciation of transforms as Node evolves.

[Example](https://github.com/sgmap/mes-aides-ui/blob/rewrite/package.json#L85-L90).